### DISCUSSION TITLE
Issue while Installing calyptia-fluentd on RHEL7

### DISCUSSION THREAD

**Discussion by swwapnilp** [fluent/fluentd#3980] [Q&A]
I'm trying to install RPM Package on the RHEL7 following the installation steps guided on this page:

I was able to install the td-agent successfully, launched the daemon, and tested it by sending sample logs via HTTP. As the next step, I tried to install "calyptia-fluentd" but while installing for Redhat using the curl command shown  below:

curl -L https://calyptia-fluentd.s3.us-east-2.amazonaws.com/calyptia-fluentd-1-redhat.sh | sh

It throws an error saying "Error: Package: calyptia-fluentd-yum-release-2021.5.1-release.noarch (/calyptia-fluentd-yum-release-2021.5.1-release.noarch)
Requires: dnf-command(config-manager)"

I have also tried to manually run the script by specifying the RedHat version but no luck. DNF-command is for RHEL 8 but not sure why it's throwing up when trying to install calyptia-fluentd on RHEL 7.

Can someone help me understand on how to resolve the issue and complete the installation? 

For ref, Please find the attached screenshot.
https://docs.fluentd.org/installation/install-by-rpm#using-to-install-calyptia-fluentd
![fluentd-error](https://user-images.githubusercontent.com/62648857/205696059-2cf62f7c-d149-4633-b7b0-091e8ac56a6d.png)

Thanks,
Swapnil Pawar

**Comment by kenhys** [comment]
Maybe missing "sudo dnf install dnf-plugin-config-manager" or something to do.

**Comment by cosmo0920** [comment]
Thanks for your report. I'd updated installation script to handle RHEL version correctly.

BTW, td-agent and also calyptia-fluentd cannot install UBI image 7.9 currently.
